docs/Exp04-Tutor.md

```diff
@@ -56,17 +56,96 @@ Now, every time you open a new terminal, you can initialize the environment of B
 BLIS_INIT
 ```
 
-### 2.3. Intel CPU
+### 2.3. MKL-Intel CPU
 
-To do.
+#### 2.3.1 Intel oneAPI Base Toolkit
+
+If you have an Intel CPU, you should install **MKL** for BLAS.
+
+It's included by **Intel oneAPI Base Toolkit**. So we'll install the toolkit. Here is the official download [page](https://www.intel.cn/content/www/cn/zh/developer/tools/oneapi/base-toolkit-download.html).
+
+Choose the `Offline/Online Installer` for any system. If your **Package Manager**'s package type is supported, you can choose it instead.
+
+The website and installer (if you choose installer) will guide you, just follow them.
+
+After installation, find `setvar.sh`. Usually, you'll find it in `~/intel/oneapi/setvars.sh` or `/opt/intel/oneapi/setvars.sh`. Source it with command:
```

Comment:
The position of `setvar.sh` is based on the commands given above (specifically with silent installation). I personally suggest to install it at default location: "/opt/intel/oneapi" for sharing with all users.